types/nunjucks/index.d.ts

```diff
@@ -114,7 +114,8 @@ export class Loader {
     emit(name: string, ...args: any[]): void;
     resolve(from: string, to: string): string;
     isRelative(filename: string): boolean;
-    extend(toExtend: ILoader): ILoader;
+    extend(extender: ILoader): ILoader;
```

Comment:
To your point, I believe this line should be removed to bring it in line with runtime behaviour?

Reply:
Sounds good, updated 👍